packages/mesos-metrics-module/buildinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "requires": ["mesos", "boost-libs", "avro-cpp"],
+  "single_source" : {
+    "kind": "git",
+    "git": "git@github.com:dcos/dcos-metrics.git",
```

Comment:
The dcos-metrics package already clones and builds this repo, having two different pkgpanda packages cloning the same repository just makes them really likely to get out of sync. Building the mesos module and building the other metrics components all of which are built from the same commit of this same repository should happen inside one pkgpanda package.